tests/sqlparser_common.rs

```diff
@@ -8685,3 +8685,15 @@ fn parse_map_access_expr() {
         let _ = dialects.verified_expr(sql);
     }
 }
+
+#[test]
+fn insert_into_with_parentheses() {
+    let dialects = TestedDialects {
+        dialects: vec![
+            Box::new(SnowflakeDialect {}),
```

Comment:
This should work with `GenericDialect` as well.

Reply:
@universalmind303 added. 
I used Redshift + Snowflake because the issue talks about these two dialects.

Reply:
Appears it was fixed in [1fce575](https://github.com/sqlparser-rs/sqlparser-rs/pull/1229/commits/1fce575ecffaaca153849ccf55e97a375fe1421b)